kubernetes/rollup-chart/files/scripts/start-geth.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -o errexit -o nounset
+
+geth --datadir $home_dir/.astriageth/ --networkid=$chain_id \
+  --http --http.addr "0.0.0.0" --http.port $executor_host_http_port --http.corsdomain='*' --http.vhosts "*" \
```

Comment:
We should add the api flags on here `--http.api eth,net,web3,debug` this enables tracing with blockscout instances